pkg/controller/replication/replication_controller.go

```diff
@@ -234,8 +234,8 @@ func (rm *ReplicationManager) Run(workers int, stopCh <-chan struct{}) {
 
 // getPodController returns the controller managing the given pod.
 // TODO: Surface that we are ignoring multiple controllers for a single pod.
-// TODO: use ownerReference.Controller to determine if the rc controls the pod.
-func (rm *ReplicationManager) getPodController(pod *api.Pod) *api.ReplicationController {
+func (rm *ReplicationManager) getPodController(pod *api.Pod) (ret_rc *api.ReplicationController) {
```

Comment:
Only use named results when you are returning multiple values of the same type.

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters